composer.json

```diff
@@ -20,8 +20,8 @@
     ],
     "require":{
         "php": ">=5.3.0",
-        "guzzle/guzzle":"3.8.1",
-        "talis/persona-php-client": "~2.0"
+        "guzzle/guzzle":">=3.8@stable",
+        "talis/persona-php-client": ">=3.1"
```

Comment:
should this  be more strict? something like `~3.1`? Doesn't `~` include all minor versions, but no major changes?